catalog/MDCCatalog/MDCCatalogTileView.swift

```diff
@@ -145,14 +146,22 @@ class MDCCatalogTileView: UIView {
       newImage = MDCDrawImage(bounds, { MDCCatalogDrawMiscTile($0, $1) }, colorScheme)
     }
 
-    guard let unwrappedImage = newImage else {
```

Comment:
for this file, i believe it would be enough to just do:
```
  override func draw(_ rect: CGRect) {
    super.draw(rect)
    imageCache.removeAllObjects()
  }
```